robottelo/cli/host.py

```diff
@@ -463,6 +463,51 @@ def sc_params(cls, options=None):
         cls.command_sub = 'sc-params'
         return cls.execute(cls._construct_command(options), output_format='csv')
 
+    @classmethod
+    def info(cls, options=None):
```

Comment:
`info` and `update` are already implemented in base class, there's bunch of tests that use it successfully right now. Why override them?

Reply:
So I saw under robottelo/cli/ I saw every sub-command having their own file and that only made sense to me to use the methods from the sub-command file. Also the methods in base requires some mandatory values like organization which is not needed in this case.

Reply:
Every sub-command has its own file, but these files usually don't override `info` or `update` methods.

If you really want to keep your implementations, then I'm going to need result for all tests that call `Host.info` or `Host.update`. We need to ensure this PR doesn't introduce regressions in existing tests.

Reply:
Using the existing methods. Not implementing the ones.